conanfile.py

```diff
@@ -81,9 +70,6 @@ def requirements(self):
                 self.requires(req)
 
     def generate(self):
-        cmake = CMakeDeps(self)
-        cmake.generate()
-
         tc = CMakeToolchain(self, generator = "Ninja")
```

Comment:
Is there a reason why you left that hard-coded value of the `generator = "Ninja"`?
We already set the global generator in the global.conf. Which is used only if this optional argument isn't provided (look at the code in the `CMakeToolchain`.

Reply:
https://github.com/Ultimaker/libnest2d/pull/8/commits